### PR TITLE
Added README.md's for Saturn and Pluto to reflect the code style appr…

### PR DESCRIPTION
**Pull request by tgreenwood**
…oach

### CODE REVIEW

File: projects/pluto/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Pluto
+
+The service is a Spring Boot service playing role of a lightweight API gateway using Spring Cloud Gateway.
```
Comment (ewelinajozwik) [inline projects/pluto/README.md]: Or 'is a Spring Boot application' to avoid 'service' repetition?

File: projects/pluto/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+### Running the app in development mode
+
+Prerequisites:
+
+- Java 21
+- Gradle
+- Running Saturn service
+- Running Keycloak instance
+- Running PostgresQL instance (optional)
+- Environment variable `KEYCLOAK_CLIENT_SECRET` set to `**********`
```
Comment (ewelinajozwik) [inline projects/pluto/README.md]: The KEYCLOAK_CLIENT_SECRET value depends on the local settings (`**********` is a default)

**Review by frankyhollywood — APPROVED** [review]

File: projects/pluto/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+### Running the app in development mode
+
+Prerequisites:
+
+- Java 21
+- Gradle
+- Running Saturn service
+- Running Keycloak instance
+- Running PostgresQL instance (optional)
```
Comment (ewelinajozwik) [inline projects/pluto/README.md]:
```suggestion
- Running PostgreSQL instance (optional)
```

File: projects/saturn/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## Development
+
+### Running the app in development mode
+
+Prerequisites:
+
+- Java 21
+- Gradle
+- PostgresQL instance running (optional)
```
Comment (ewelinajozwik) [inline projects/saturn/README.md]:
```suggestion
- PostgreSQL instance running (optional)
```

File: projects/saturn/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+- Keycloak instance running
```
Comment (ewelinajozwik) [inline projects/saturn/README.md]: `KEYCLOAK_CLIENT_SECRET` is also needed (same as for pluto)

File: projects/saturn/README.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+To run the application, execute the main method in the [App](src/main/java/io/fairspace/saturn/App.java) class or execute the following command:
```
Comment (ewelinajozwik) [inline projects/saturn/README.md]: I would mention the command only, otherwise we should specify gradle setup of the IDE.

Comment (tgreenwood) [reply projects/saturn/README.md]: good points, updated according all suggestions

File: projects/saturn/README.md (hunk 5 above)
Comment (tgreenwood) [reply projects/saturn/README.md]: updated

File: projects/pluto/README.md (hunk 2 above)
Comment (tgreenwood) [reply projects/pluto/README.md]: updated

File: projects/pluto/README.md (hunk 1 above)
Comment (tgreenwood) [reply projects/pluto/README.md]: updated

**Review by ewelinajozwik — APPROVED** [review]

**Pull request merged**